nucypher/blockchain/eth/sol/__init__.py

```diff
@@ -14,3 +14,5 @@
 You should have received a copy of the GNU Affero General Public License
 along with nucypher.  If not, see <https://www.gnu.org/licenses/>.
 """
+
+SOLIDITY_COMPILER_VERSION = '0.5.16'
```

Comment:
This is the canonical version - correct?

Reply:
yep, https://github.com/ethereum/solidity/releases/tag/v0.5.16